multi-agent/tests/e2e/test_session_stress.py

```
@@ -810,29 +1030,39 @@ def test_concurrent_session_creation_and_execution(
+                    print(f"Deleting {len(session_ids)} sessions...")
+                    deleted_count = 0
+                    for session_id in session_ids:
+                        if api_client.delete_session(session_id):
+                            deleted_count += 1
+                    print(f"  ✅ Deleted {deleted_count}/{len(session_ids)} sessions")
+
+                # Only delete blueprints this test created
+                if blueprint_id and owns_blueprint:
+                    print(f"Deleting blueprint {blueprint_id[:8]}...")
+                    if api_client.delete_blueprint(blueprint_id):
+                        print(f"  ✅ Blueprint deleted")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)